server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/json/SessionSerializationTest.scala

```diff
@@ -87,28 +85,32 @@ object SessionSerializationTest {
   private val IS_NOT_PERSONAL : IsPersonal = IsPersonal(false)
   private val IS_NOT_READ_ONLY : IsReadOnly = IsReadOnly(false)
 
-  private val ACCOUNT_1 = Account(
+  private val ACCOUNT_1: Account = Account.from(
     name = USER_1,
     isPersonal = IS_PERSONAL,
     isReadOnly = IS_NOT_READ_ONLY,
-    accountCapabilities = Set(CORE_CAPABILITY))
-  private val ACCOUNT_2 = Account(
+    accountCapabilities = Set(CORE_CAPABILITY)) match {
```

Comment:
you can use `map`

Reply:
`map` just gives me another `Either`

Reply:
I will use `.toOption.get` like what Remi said